tests/components/shelly/test_climate.py

```diff
@@ -560,7 +560,7 @@ async def test_device_not_calibrated(
     hass: HomeAssistant,
     mock_block_device: Mock,
     monkeypatch: pytest.MonkeyPatch,
-    issue_registry: IssueRegistry,
+    issue_registry: ir.IssueRegistry,
```

Comment:
What is the benefit in using this kind of import? Looking at all the other import we import just the class we use, what is the difference in this case?

Reply:
There is no benefit, but is standardize across our eco system (see also the definition of all registries in `pyproject.toml`

Reply:
We have such short namespaces for area/device/entity/issue registries.
It is mostly to standardize, and avoid such things as:
`from homeassistant.helpers.issue_registry import async_get as async_get_issue_registry`